### PR TITLE
D331

### PR DESCRIPTION
**Pull request by valerii-fed**
Default description

### CODE REVIEW

File: first_file.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ line6
 line7
```
Comment (valerii-fed) [inline first_file.txt]: Author: valerii.fedorenko.ua@gmail.com
Created time: 2018-11-07T16:32:42.815Z
Body: R 7
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/331/test-repo3/first_file.txt

File: first_file.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@ line6
...
 line8
 line9
-line10
+CHANGED_line10
 line11
 line12
 line13
```
Comment (valerii-fed) [inline first_file.txt]: Author: valerii.fedorenko.ua@gmail.com
Created time: 2018-11-07T16:41:19.036Z
Body: R 13
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/331/test-repo3/first_file.txt

File: first_file.txt [hunk 3]
```diff
@@ -7,7 +7,7 @@ line6
 line7
 line8
 line9
-line10
+CHANGED_line10
```
Comment (valerii-fed) [inline first_file.txt]: Author: valerii.fedorenko.ua@gmail.com
Created time: 2018-11-14T08:48:25.793Z
Body: R 10
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/331/test-repo3/first_file.txt

File: first_file.txt [hunk 4]
```diff
@@ -7,7 +7,7 @@ line6
...
 line11
 line12
```
Comment (val-fed) [inline first_file.txt]: L 12 GitHub

File: first_file.txt (hunk 1 above)
Comment (valerii-fed) [inline first_file.txt]: Author: valerii.fedorenko.ua@gmail.com
Created time: 2018-11-07T16:32:42.815Z
Body: R 7
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/331/test-repo3/first_file.txt

Comment (valerii-fed) [inline first_file.txt]: Author: valerii.fedorenko.ua@gmail.com
Created time: 2018-11-07T16:32:42.815Z
Body: R 7
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/331/test-repo3/first_file.txt

Comment (valerii-fed) [inline first_file.txt]: Author: valerii.fedorenko.ua@gmail.com
Created time: 2018-11-07T16:32:42.815Z
Body: R 7
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/331/test-repo3/first_file.txt

Comment (valerii-fed) [inline first_file.txt]: Author: valerii.fedorenko.ua@gmail.com
Created time: 2018-11-07T16:32:42.815Z
Body: R 7
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/331/test-repo3/first_file.txt

**Pull request closed without merging**